From b76741fd09aadce7797cb07b466b11b24b0c3225 Mon Sep 17 00:00:00 2001
From: David Cohen <david.a.cohen@intel.com>
Date: Mon, 9 Jun 2014 15:49:20 -0700
Subject: [PATCH 2/3] pmic: dollar_cove: HACK: do not handle otg_gpio

MRD7's phy driver will handle otg_gpio (as well as mux and id gpios) as
a short term solution to enable usb device and provide initial support
of dual role.

Signed-off-by: David Cohen <david.a.cohen@intel.com>
---
 drivers/mfd/intel_soc_pmic_dc.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

Index: linux/drivers/mfd/intel_soc_pmic_dc.c
===================================================================
--- linux.orig/drivers/mfd/intel_soc_pmic_dc.c	2014-06-23 12:02:56.067284542 -0700
+++ linux/drivers/mfd/intel_soc_pmic_dc.c	2014-06-23 12:04:12.687174802 -0700
@@ -363,14 +363,9 @@
 	pdata.def_iterm = 300;
 	pdata.def_max_temp = 55;
 	pdata.def_min_temp = 0;
-
-	pdata.otg_gpio = 117; /* GPIONC_15 */
-	/* configure output */
-	ret = gpio_request(pdata.otg_gpio, "otg_gpio");
-	if (ret) {
-		pr_err("unable to request GPIO pin\n");
-		pdata.otg_gpio = -1;
-	}
+	
+	/* Deprecated: DC does not handle GPIO for VBUS */
+	pdata.otg_gpio = -1;
 
 	platform_init_chrg_params(&pdata);
 
